person.rb

```diff
@@ -21,6 +23,10 @@ def correct_name
     name
   end
 
+  def add_rental(book, date)
+    Rental.new(date, self, book)
```

Comment:
- [ ] Kindly note the order of the arguments as requested by the previous reviewer. Your rental class expects the date, book, and person in that order, but here you have date, person (self) and book. Kindly fix that. It is a little but quite significant error.

  ```
  Rental.new(date, book, self)
  ```